lib/eml2html/converter.rb

```diff
@@ -55,6 +56,33 @@ def save_zip(options = {})
       end
     end
 
+    def get_attachments(include_in_doc=false)
+      attachments = []
```

Comment:
Please don't call methods like `get_`. Please call it `attachments`

```ruby
return @attachments if include_in_doc

@attachments.reject do |attachment|
  @cids_in_body.include?(a.cid)
end
```

Reply:
done